client/src/modules/list/components/Items/VotingBox/VotingBox.jsx

```diff
@@ -1,34 +1,48 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 import ThumbIcon from 'assets/images/thumbs-up-solid.svg';
 
-const VotingBox = ({ isVoted, voteForItem, votesCount }) => (
-  <button
-    className={classNames('voting-box', {
-      'voting-box__voted': isVoted
-    })}
-    onClick={e => {
-      e.stopPropagation();
+class VotingBox extends PureComponent {
+  handleOnClick = event => {
+    event.stopPropagation();
+    const { isMember, voteForItem } = this.props;
+
+    if (isMember) {
       voteForItem();
-    }}
-    type="button"
-  >
-    <img
-      alt={`${isVoted ? 'remove your vote' : 'vote'}`}
-      className="voting-box__icon"
-      src={ThumbIcon}
-    />
-    {votesCount}
-  </button>
-);
+    }
+  };
+
+  render() {
+    const { isMember, isVoted, votesCount } = this.props;
+
+    return (
+      <button
+        className={classNames('voting-box', {
+          'voting-box__voted': isVoted,
+          'voting-box__disabled': !isMember
+        })}
+        onClick={this.handleOnClick}
```

Comment:
disable invoking handleOnClick and you wouldn't have to check isMember inside that handler